backend/src/main/java/ca/gov/dtsstn/cdcp/api/controller/SubscriptionController.java

```diff
@@ -0,0 +1,45 @@
+package ca.gov.dtsstn.cdcp.api.controller;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import ca.gov.dtsstn.cdcp.api.dto.ImmutableSubscription;
+
+@RestController
+@RequestMapping(path={"/v1/user"})
+public class SubscriptionController {
+
+	@GetMapping("{userId}/subscriptions")
+	public List<ImmutableSubscription> getSubscriptions(@PathVariable String userId) {
+		List<ImmutableSubscription> subscriptions= new ArrayList<ImmutableSubscription>();
+		ImmutableSubscription aSubscription = ImmutableSubscription.builder()
+			.id(10001L)
+			.sin("800011819")
+			.email("user@example.com")
+			.registered(true)
+			.subscribed(true)
+			.preferredLanguage(1033L)
+			.alertType("cdcp")
+			.build();
+
+		subscriptions.add(aSubscription);
+		aSubscription = ImmutableSubscription.builder()
+			.id(10002L)
+			.sin("800011819")
+			.email("user@example.com")
+			.registered(true)
+			.subscribed(false)
+			.preferredLanguage(1033L)
+			.alertType("ei")
+			.build();
+		subscriptions.add(aSubscription);
```

Comment:
Consider using `List.of(..)` here..

ex:

``` java
return List.of(
	ImmutableSubscription.builder()
		...
		.build(),
	ImmutableSubscription.builder()
		...
		.build())
```

Reply:
fixed